thread_numbers: 4

# Parameters for control loop
pub_twist_cmd: true
debug_info: true
delay_mode: false
max_speed: 4.0 # unit: m/s
waypoints_dist: -1.0 # unit: m, set < 0 means computed by node
path_length: 2.0 # unit: m
goal_radius: 0.2 # unit: m
controller_freq: 10
vehicle_Lf: 0.2 # distance between the front of the vehicle and its center of gravity

# Parameter for MPC solver
mpc_steps: 20.0
mpc_ref_cte: 0.0
mpc_ref_epsi: 0.0
mpc_ref_vel: 2.0
mpc_w_cte: 100.0
mpc_w_epsi: 100.0
mpc_w_vel: 100.0
mpc_w_delta: 100.0
mpc_w_accel: 50.0
mpc_w_delta_d: 0.0
mpc_w_accel_d: 0.0
mpc_max_steering: 0.836 # Maximal steering radian (~25 deg)
mpc_max_throttle: 0.8 # Maximal throttle accel
mpc_bound_value: 1.0e3 # Bound value for other variables